src/Apps/Artist/Routes/Works/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { Box, Col, Row, Separator } from "@artsy/palette"
+import { Works_artist } from "__generated__/Works_artist.graphql"
+import { ArtistCollectionsRailContent as ArtistCollectionsRail } from "Apps/Artist/Components/ArtistCollectionsRail"
+import { ArtistArtworkFilterRefetchContainer as ArtworkFilter } from "Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter"
+import { ArtistRecommendationsQueryRenderer as ArtistRecommendations } from "Apps/Artist/Routes/Overview/Components/ArtistRecommendations"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { get } from "Utils/get"
+
+export interface WorksRouteProps {
+  artist: Works_artist
+}
+
+export const WorksRoute: React.FC<WorksRouteProps> = props => {
+  const { artist } = props
+  const { sidebarAggregations } = artist
+
+  const isClient = typeof window !== "undefined"
```

Comment:
This isn't great since it hardcodes `window`, isn't portable to RN, etc.

Instead, should use a `componentDidMount` if a class method, or a hook otherwise.

Reply:
👍this follows existing patterns in the artist app so will follow up afterwards and fix other instances too.